src/utils/payments/pawapay/types.ts

```
@@ -0,0 +1,69 @@
+// Define interfaces for request and response types
+interface PaymentPageSessionRequest {
```

ESLint

[error] Use a `type` instead of an `interface`.

(ts/consistent-type-definitions)